src/elements/Payment.vue

```diff
@@ -129,19 +129,31 @@ export default {
       try {
         this.$emit('loading', true);
         event.preventDefault();
-        const { error } = await this.stripe.confirmPayment({
+        const response = await this.stripe.confirmPayment({
           elements: this.elements,
           confirmParams: this.confirmParams,
           redirect: this.redirect
         });
-        if (error) {
+
+        // if the response is an error
+        if ( response.error ) {
           const errorElement = document.getElementById(
             'stripe-payment-element-errors',
           );
-          errorElement.textContent = error.message;
-          this.$emit('error', error);
+          errorElement.textContent = response.error.message;
+          this.$emit('error', response.error);
           return;
+
+        } else if( response.paymentIntent ) {
```

Comment:
Please run through the linter. There should be a space between `if` and `(`. Thank you.